### PR TITLE
Feature/update Gui & Fixing no Iso erros

### PR DESCRIPTION
**Pull request by mrrfr**
=> Removing NewSimulation Twice Call.
=> Adding a Constant for people's distress
=> Adding a graveyard
=> Working only with Sim.Persons instead of using

### CODE REVIEW

**Comment by SAIDI-NASSIM** [comment]
lgtm

**Comment by gmiralle** [comment]
lgtm

File: pkg/simulation/map.go [hunk 1]
```diff
@@ -20,10 +20,10 @@ type MapCell struct {
...
 // Map represents the entire simulation environment
 type Map struct {
-	Width     int
-	Height    int
-	Persons   []*persons.Person
-	Drones    []*drones.Drone
+	Width  int
+	Height int
+	//Persons   []*persons.Person
+	//Drones    []*drones.Drone
```
Comment (mrrfr) [inline pkg/simulation/map.go:26]: Are u Sure we want to delete this ?

**Review by mrrfr — COMMENTED** [review]
Lgtm, but not sure for this part, wanted ur advices.

File: pkg/simulation/map.go (hunk 1 above)
Comment (gmiralle) [reply pkg/simulation/map.go:26]: grave bonne idée frr

Comment (SAIDI-NASSIM) [reply pkg/simulation/map.go:26]: stfu

**Pull request merged**